diff --git a/es/index.js b/es/index.js
index a62cc20bf62d7accd444f873e09aaf053bb5c00b..e48f172da9ab22f6768eb722be4fa0961acba96a 100644
--- a/es/index.js
+++ b/es/index.js
@@ -722,4 +722,4 @@ var shouldNavigate = function shouldNavigate(event) {
 };
 
 ////////////////////////////////////////////////////////////////////////
-export { Link, Location, LocationProvider, Match, Redirect, Router, ServerLocation, createHistory, createMemorySource, isRedirect, navigate, redirectTo, globalHistory, match as matchPath, useLocation, useNavigate, useParams, useMatch, resolve };
\ No newline at end of file
+export { Link, Location, LocationProvider, Match, Redirect, Router, ServerLocation, createHistory, createMemorySource, isRedirect, navigate, redirectTo, globalHistory, match as matchPath, useLocation, useNavigate, useParams, useMatch, resolve, BaseContext };
diff --git a/index.js b/index.js
index f31a4e043a917cf1e4a7e757b6cf0f90e4212f60..6146e75397e681b7a22feffaac494891a1376d99 100644
--- a/index.js
+++ b/index.js
@@ -759,3 +759,4 @@ exports.useNavigate = useNavigate;
 exports.useParams = useParams;
 exports.useMatch = useMatch;
 exports.resolve = _utils.resolve;
+exports.BaseContext = BaseContext;
